### DISCUSSION TITLE
Beta FPV Super P 14ch PWM Rx. Future GPS support?

### DISCUSSION THREAD

**Discussion by davidabevan** [ExpressLRS/ExpressLRS#2606] [Q&A]
Hello ELRS Devs and users.
Am wondering if use of a GPS eg bn220 etc
Over the crsf is possible now or as a future feature maybe similar to the matek PWM vario Rx. So basically a standalone GPS  port/Rx uart for fixed wing with out a flight controller.
Thanks in advance 🙂

**Comment by DabblenBabble** [comment]
Sorry to see no responses here... I'm about to try this out myself later this week!  I imagine it would work...

**Comment by Mkorho** [comment]
Standalone GPS modules are supported in 4.0
https://github.com/ExpressLRS/ExpressLRS/pull/3086

**Reply by DabblenBabble** [reply]
I'm afraid I'm pretty new to this stuff... are you saying GPS support is added in ELRS 4.0, as in a full version newer than 3.3, or some other 4.0?
